docs/index.rst

```diff
@@ -23,10 +23,14 @@
    man/terminology.rst
 
 
-######################################################
+#########################################
 WEC-Sim (Wave Energy Converter SIMulator) 
-######################################################
-`WEC-Sim (Wave Energy Converter SIMulator) <https://github.com/WEC-Sim/WEC-Sim>`_ is an open-source code for simulating wave energy converters. The code is developed in MATLAB/SIMULINK using the multi-body dynamics solver Simscape Multibody. WEC-Sim has the ability to model devices that are comprised of bodies, joints, power take-off systems, and mooring systems. WEC-Sim can model both rigid and flexible with generalized body modes. Simulations are performed in the time-domain by solving the governing wave energy converter equations of motion in the 6 Cartesian degrees-of-freedom, plus any number of user-defined modes. The `WEC-Sim Applications repository <https://github.com/WEC-Sim/WEC-Sim_Applications>`_ contains a wide variety of scenarios that WEC-Sim can be used to model, including desalination, mooring dynamics, nonlinear hydrodynamic bodies, passive yawing, batch simulations and many others. The code is very flexible and can be adapted to many scenarios within the wave energy industry.
+#########################################
+
+`WEC-Sim (Wave Energy Converter SIMulator) <https://github.com/WEC-Sim/WEC-Sim>`__ 
+is an open-source code for simulating the energy production, forces and motions 
+of wave energy converters. 
```

Comment:
This seems a bit limiting on WEC-Sim's capabilities up front. I like the idea of expanding on this in a new section like you propose in issue #455 (Introduction/Overview). There the docs can dive into more detail on WEC-Sim's capabilities and give examples.

Reply:
I would prefer to keep the WEC-Sim description was it was, but agree with @akeeste about expanding upon this in a new section.

Reply:
I reverted this change